startup/gui/editor.py

```diff
@@ -45,28 +45,6 @@ def __editorCreated( editor ) :
 
 GafferUI.Editor.instanceCreatedSignal().connect( __editorCreated, scoped = False )
 
-
-#### Editor follows Scene Selection
-
-# This makes the supplied editor follow the source node of the scene selection of another editor.
-# When followSelectionSource is True, the editor will follow the source node for the scene
-# selection (as observed by the targetEditor's nodeSet), instead of the targets nodeSet itself.
-# For example, a NodeEditor linked to a Viewer with followSelectionSource would show the source
-# node for any selected lights/objects.
-
-def __driveFromSceneSelectionSourceChangeCallback( editor, targetEditor ) :
-
-	sourceSet = targetEditor.getNodeSet()
-	context = targetEditor.getContext()
-	return GafferSceneUI.SourceSet( context, sourceSet )
```

Comment:
Let's also remove SourceSet, since it doesn't have any uses outside of this feature. I think there's probably also an unused icon in `graphics.svg` that we could remove now.

Reply:
Good idea, removed [here](https://github.com/GafferHQ/gaffer/pull/4085/commits/9f8f2229975faf894a4cc3fb6f340f1706dea11f).